io-ballerina/src/io/tests/unit-tests/bytes_io.bal

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+
+ReadableByteChannel? bytesReadCh = ();
+WritableByteChannel? bytesWriteCh = ();
+
+function testReadBytes() {
```

Comment:
Don't we need the `@test:Config{}` annotation for these test cases?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/1/commits/ffea910e91926ad14aa6750ec050430ac157f658

Reply:
Still, there are print-related fixes are remaining, we need to figure it out a way to test those using testerina.

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/1/commits/59b655b325f77b764bf2648e93192ed29d5b1a83